locust/env.py

```diff
@@ -94,6 +92,12 @@ def __init__(
         """
         self.parsed_options = parsed_options
         """Reference to the parsed command line options (used to pre-populate fields in Web UI). When using Locust as a library, this should either be `None` or an object created by `argument_parser.parse_args()`"""
+        self.exclude_failed_response_time = False
+        if self.parsed_options and self.parsed_options.exclude_failed_response_time:
+            self.exclude_failed_response_time = True
+        self.stats = RequestStats(exclude_failed_response_time=self.exclude_failed_response_time)
+        """Reference to RequestStats instance"""
```

Comment:
I dont understand this comment.

Reply:
I did not write it, this is probably a copy paste issue for restructuring.